velox/exec/HashJoinBridge.h

```diff
@@ -174,9 +174,10 @@ bool isLeftNullAwareJoinWithFilter(
 class HashJoinMemoryReclaimer final : public MemoryReclaimer {
  public:
   static std::unique_ptr<memory::MemoryReclaimer> create(
-      std::shared_ptr<HashJoinBridge> joinBridge) {
+      std::shared_ptr<HashJoinBridge> joinBridge,
+      int32_t priority = 0) {
```

Comment:
Do we enforce a priority input later?

Reply:
Yeah we can probably enforce that later. But now we don't want to enforce as the current priority setting is on Task level.